apps/explorer/src/pages/transaction-result/transaction-summary/ObjectChanges.tsx

```diff
@@ -0,0 +1,370 @@
+                            renderFooter && (
+                                <div className="flex items-center justify-between">
+                                    <Text
+                                        variant="pBody/medium"
+                                        color="steel-dark"
+                                    >
+                                        Owner
+                                    </Text>
+                                    {locationIdType ===
+                                        LocationIdType.AddressOwner && (
+                                        <AddressLink address={ownerAddress} />
+                                    )}
+                                    {locationIdType ===
+                                        LocationIdType.ObjectOwner && (
+                                        <ObjectLink objectId={ownerAddress} />
+                                    )}
+                                    {locationIdType ===
+                                        LocationIdType.Shared && (
+                                        <ObjectLink
+                                            objectId={ownerAddress}
+                                            label="Shared"
+                                        />
+                                    )}
+                                </div>
+                            )
+                        }
+                    >
+                        <TransactionCardSection
+                            title={
+                                <Text
+                                    variant="body/semibold"
+                                    color="steel-darker"
+                                >
+                                    {title}
+                                </Text>
+                            }
+                        >
+                            <ExpandableList
+                                items={changes.map(
+                                    ({ objectId, objectType }) => (
+                                        <ObjectDetail
+                                            key={objectId}
+                                            objectId={objectId}
+                                            objectType={objectType}
+                                        />
+                                    )
+                                )}
+                                defaultItemsToShow={DEFAULT_ITEMS_TO_SHOW}
+                                itemsLabel="Objects"
+                            >
+                                <div className={clsx(expandableListClassName)}>
+                                    <ExpandableListItems />
+                                </div>
+                            </ExpandableList>
+                        </TransactionCardSection>
+                    </TransactionCard>
+                );
+            })}
+        </>
+    );
+}
+
+function groupByOwner(
```

Comment:
Why isn't this just being done in the core hook so that it's consistent between wallet and explorer?